src/utils/yup.js

```diff
@@ -0,0 +1,21 @@
+import { setLocale } from "yup";
+import * as yup from "yup";
+
+setLocale({
```

Comment:
Maybe it would be good to apply these locales globally for the app? For example import the file containing the in `main.jsx` to apply the i18n for all the forms using `yup`?